pkg/agent/openflow/packetin.go

```diff
@@ -53,6 +53,8 @@ const (
 	// PacketInCategorySvcReject is used to process the Service packets not matching any
 	// Endpoints within packetIn message.
 	PacketInCategorySvcReject
+	// PacketInCategoryPC is used for packetIn messages related to capture.
+	PacketInCategoryPC
```

Comment:
nit: can we just use `PacketInCategoryPacketCapture`, it's more readable?

Reply:
sure